### ISSUE TITLE
Create Column Names Using Generics

### ISSUE THREAD

**Issue by jedielson** [nicolastakashi/flepper#23]
What about an extension method to create the column names to be used on Select Command using Generics?

I was wondering if could be somethng like this!

``` csharp
public static ISelectCommand Select<T>() where T : class
{
       // code
}
```

With some reflection we can get all properties of DTO and build the column names!

**Comment by nicolastakashi** [comment]
It's looks good to me!
We can put it into a backlog item and mark this issue as a feature, to implement it. You can wait a team member do it or if you want you can send a pull request to us

**Comment by jedielson** [comment]
Ok, I'll proceed with it, and submit a PR!

**Comment by gustavoferrazfontes** [comment]
For performance reasons, would be nice has a cache to save the properties name. Then once a type is requested, we can get by cache.

**Comment by nicolastakashi** [comment]
I like so much a cache idea, we can use ConcurrentDictionary

**Comment by gustavoferrazfontes** [comment]
This issue was implemented in PR #24.  Thank you!
